wiki/Community/How_you_can_help!/id.md

```diff
@@ -1,74 +1,74 @@
-# Bagaimana Kamu dapat membantu!
+# Bagaimana kamu dapat membantu!
 
-osu! memiliki komunitas yang sangat dinamis yang terdiri dari orang-orang dari seluruh dunia. Dan siapa pun dapat menjadi bagian dari komunitas dengan caranya sendiri!
+osu! memiliki komunitas yang sangat dinamis yang terdiri dari orang-orang dari seluruh dunia. Siapa pun dapat menjadi bagian dari komunitas dengan caranya sendiri!
 
-Di bawah ini hanyalah beberapa cara yang dapat Kamu lakukan untuk membantu dalam komunitas:
+Di bawah ini merupakan beberapa cara yang dapat kamu lakukan untuk membantu dalam komunitas:
 
-## Membantu dalam wiki
+## Berkontribusi pada wiki
 
 [Wiki](https://github.com/ppy/osu-wiki) adalah basis pengetahuan resmi kami yang berisi apa saja dan segalanya tentang osu!. Oleh karena itu, wiki selalu membutuhkan lebih banyak kontributor untuk membantu menjaga situs tetap mutakhir dengan pengetahuan maupun terjemahan yang segar dan baru.
 
-Jika Kamu adalah pendatang baru di osu! wiki, Kamu dapat membaca lebih lanjut di [Panduan Kontribusi osu! wiki](/wiki/osu!_wiki/Contribution_guide).
+Jika kamu adalah pendatang baru di osu! wiki, kamu dapat membaca lebih lanjut di [Panduan kontribusi osu! wiki](/wiki/osu!_wiki/Contribution_guide).
 
 ## Membantu menjaga osu! tetap ramah dan aman
 
 Untuk menjaga keamanan hub komunitas resmi, osu! memiliki tim moderasi khusus yang memantau forum dan obrolan online sepanjang waktu.
 
-Untungnya, masih mudah bagi Kamu untuk membantu menjaga semuanya tetap aman. Berikut ini adalah sebagian kecil dari banyak cara yang dapat Kamu lakukan untuk membantu:
+Untungnya, masih mudah bagi kamu untuk membantu menjaga semuanya tetap aman. Berikut ini adalah sebagian kecil dari banyak cara yang dapat kamu lakukan untuk membantu:
 
-- Ikuti kanal favorit Kamu di [obrolan osu! dalam gim](/wiki/Chat_Console), atau di forum.
-- Jika Kamu melihat suatu peristiwa yang tidak pantas atau mengganggu, gunakan perintah `!report` dalam obrolan dalam osu! untuk memberi tahu moderator.
+- Ikuti *channel* favorit kamu di [obrolan osu! dalam permainan](/wiki/Chat_Console) atau di forum.
+- Jika kamu melihat suatu peristiwa yang tidak pantas atau mengganggu, gunakan perintah `!report` dalam obrolan dalam osu! untuk memberi tahu moderator.
```

Comment:
link to `/wiki/Reporting_Bad_Behaviour`

![image](https://user-images.githubusercontent.com/36758269/145217037-674df408-a089-4330-b7f2-db606fc10ad1.png)

Reply:
Thank you, I also noticed there are terms that need to be linked